pipelines/docker-build-rhtap/patch.yaml

```diff
@@ -205,8 +205,10 @@
       - update-deployment
     when:
       - input: "$(params.event-type)"
-        operator: notin
-        values: ["pull_request", "Merge Request"]
+        operator: in
+        values:
```

Comment:
Do we need a value in the list for incoming request (pipeline rerun?)

Reply:
I didn't find a specific event type for that, let me check again

Reply:
From the [PaC docs here](https://pipelinesascode.com/docs/guide/incoming_webhook/):
```
kind: PipelineRun
metadata:
  annotations:
    pipelinesascode.tekton.dev/on-event: "[incoming]"
```

Reply:
I'm not sure if RHTAP uses it. With exclude PR events it was theoretically working.

Reply:
True... well, let's see what happens